src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -317,6 +317,46 @@ public function symlink($originDir, $targetDir, $copyOnWindows = false)
     }
 
     /**
+     * Return the target of a link.
+     *
+     * @param string $path        A link path.
+     * @param bool   $finalTarget Whether or not to recursively follow links
+     *                            until a final target is reached.
+     *
+     * @return string Return the resolved link.
+     *
+     * @throws IOException When the link does not exist or is not readable.
+     */
+    public function readlink($path, $finalTarget = false)
```

Comment:
When you say realpath, what do you mean :) ? With final target to true ?

We are using both the "not final" and the "final" cases in Puli, I think two methods is a good idea indeed. How should we name them though? If we choose realpath/readlink, which convention do we choose? I would prefer the Unix one personnally (realpath = final, readlink = not final).